### ISSUE TITLE
type mismatch in rpcgen.cpp

### ISSUE THREAD

**Issue by frmeier** [frmeier/psi46test#5]
In rpcgen/rpcgen.cpp there is a type mismatch on line 129:

```
129: fprintf(f, "const uint16_t rpc_cmdListSize = %lu;\n\n", (unsigned int)(fl.size()));
```

which generates a warning. Is this done on purpose or is it an error?

**Comment by meierb** [comment]
I think, it's an error. Can you correct it Frank?

**Comment by frmeier** [comment]
Fixed by commit 3f9afee...
